nltk/embedding/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Natural Language Toolkit: word embedding package
+#
+# Copyright (C) 2001-2015 NLTK Project
+# Author: Long Duong <longdt219@gmail.com>
+# URL: <http://nltk.org/>
+# For license information, see LICENSE.TXT
+
+# Import Gensim if installed
+try:
```

Comment:
I like extras_require, +1 to add it. Maybe we shouldn't call it "everything" because many NLTK dependencies can't be installed with pip as they're non-Python.